packages/loader/driver-utils/src/network.ts

```diff
@@ -193,7 +193,8 @@ export function createGenericNetworkError(
  * I.e. asserts or unexpected exceptions in our code result in container failure.
  * @param error - The error to inspect for ability to retry
  */
-export const canRetryOnError = (error: any): boolean => error?.canRetry === true;
+export const canRetryOnError = (error: any): boolean =>
+	error?.canRetry === true || error?.forceRetry === true;
```

Comment:
still reading, but i'm prefer it we didn't have to modify this

Reply:
We could modify the RunWithRetry method instead of adding forceRetry to canRetryOnError